src/treq/client.py

```diff
@@ -377,7 +364,7 @@ def _guess_content_type(filename):
 registerAdapter(_from_bytes, bytes, IBodyProducer)
 registerAdapter(_from_file, BytesIO, IBodyProducer)
 
-if not _PY3:
+if not PY3:
     from StringIO import StringIO
     registerAdapter(_from_file, StringIO, IBodyProducer)
     # Suppress lint failure on Python 3.
```

Comment:
this else imports `io` late, hoist it to the top, as we already depend on `io.BytesIO`